README.md

```diff
@@ -39,6 +39,10 @@ These functions operate on `wad` only.
 - `floor(n)` - Round a value down to the nearest `wad`
 - `ceil(n)` - Round a value up to the nearest `wad`
 
+These functions operate on any `ufelt` values in the range [0, 2<sup>125</sup>], including `wad` and `ray`.
+- `unsigned_min(n)` - Returns the smaller of two values
+- `unsigned_max(n)` - Returns the larger of two values
```

Comment:
min/max take 2 params :)

Reply:
Ah time to break for the day, thanks again!